src/bin/lib/render-migration.ts

```diff
@@ -28,28 +28,28 @@ const renderPlan = (batches: RequestBatch[]) => {
     }
 
     for (const error of batch.validationErrors) {
-      console.log(chalk`{red.bold Error: ${error.message}}`)
+      console.error(chalk`{red.bold Error: ${error.message}}`)
     }
 
     if (batch.intent.shouldPublish()) {
-      console.log(chalk`\n{bold.underline Publish Content Type} {bold.yellow ${batch.intent.getContentTypeId()}}`)
+      console.error(chalk`\n{bold.underline Publish Content Type} {bold.yellow ${batch.intent.getContentTypeId()}}`)
```

Comment:
This isn't an error case. It's part of the normal plan output.

Reply:
Updated! I assumed that the `yellow` color inferred warning/error in this case